eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/SlugConverter.php

```diff
@@ -74,7 +73,6 @@ class SlugConverter
                     'endline_search_normalize',
                     'tab_search_normalize',
                     'specialwords_search_normalize',
-                    'punctuation_normalize',
```

Comment:
@arnebratt "need" was not in my statement :) I said no change, by that I meant fuctionality is not changed, as far as I can tell. The code is changed as you say - I understood @andrerom as: If punctuation_normalize is wrong for urlalias, then it is also wrong for urlalias_compat, whether or not the change has any apparent practical effect at the moment.